pkg/controller/common/util.go

```diff
@@ -612,6 +618,12 @@ func GetPodsUsingPVCs(ctx context.Context, c client.Client, namespace string, na
 						// all mounts must be ro
 						addPod = false
 					}
+					if strings.HasSuffix(pod.Name, common.ClonerSourcePodNameSuffix) && pod.Labels != nil &&
+						pod.Labels[common.CDIComponentLabel] == common.ClonerSourcePodName {
```

Comment:
`pod.Labels != nil` is not required.   `pod.Labels[common.CDIComponentLabel] == common.ClonerSourcePodName` can still be executed when pod.Labels is nil